error compiling Upload component

- [ x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[http://none.com](http://none.com)

### Steps to reproduce
`yarn tsc`

### What is expected?
compiles without issues

### What is actually happening?
getting this error

```
$ tsc -v && tsc
Version 3.9.2
node_modules/antd/lib/upload/Upload.d.ts:2:22 - error TS7016: Could not find a declaration file for module 'rc-upload'. 'D:/adcode/dashboard/node_modules/rc-upload/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/rc-upload` if it exists or add a new declaration (.d.ts) file containing `declare module 'rc-upload';`

2 import RcUpload from 'rc-upload';
                       ~~~~~~~~~~~


Found 1 error.

error Command failed with exit code 2.
```

| Environment | Info |
|---|---|
| antd | 4.2.3 |
| React | 16.13.1 |
| System | windows 10 |
| Browser | chrome |


<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

--- a/components/upload/Upload.tsx
+++ b/components/upload/Upload.tsx
@@ -72,7 +72,7 @@ class Upload extends React.Component<UploadProps, UploadState> {
     this.clearProgressTimer();
   }
 
-  saveUpload = (node: typeof RcUpload) => {
+  saveUpload = (node: any) => {
     this.upload = node;
   };
 

